frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groupActionMenu/groupSettingsForm/groupSettingsForm.component.tsx

```diff
@@ -132,7 +132,7 @@ export const GroupSettingsForm = ({ value, onSubmit, onCancel, prefixes, isColor
 			newValues.group.objects = meshObjectsToV5GroupNode(data);
 		}
 
-		if (!newValues.color) {
+		if (!isColored) {
```

Comment:
This creates an error when the user selects the no-color option (the one with a red diagonal line)